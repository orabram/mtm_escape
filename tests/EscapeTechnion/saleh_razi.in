#bad 2  invalid 
#1)Error: The parameter sent to the function is invalid
#2)Error: The parameter sent to the function is invalid
company add razi@@ 5
company add razi@  18

#Good
company add company1@ 5
company add company2@ 5
company add company3@ 5
company add company4@ 9
company add company5@ 9
company add company6@ 9
company add company7@ 10

#bad  
#1)Error: The email already exists in the system
#2)Error: The email already exists in the system

company add company3@ 5
company add company3@ 10

#bad  
#1)Error: The email entered does not belong to any company in the system
#2)Error: The parameter sent to the function is invalid

room add company8@  5 40 10 08-15 4
room add company5@  0 40 10 08-15 4

#good rooms
#        email     id-price num_ppl working difficulty
room add company1@ 1  40     3      15-22 5
room add company1@ 2  20     2           12-19 7
room add company2@ 3  28     1          22-24 4
room add company3@ 4  60     4            00-07 9
room add company3@ 5  16     10      12-20  6
room add company4@ 6  24     4    10-15   3
room add company4@ 7  40     3    16-23   4   
room add company5@ 8  36     2    19-24    5
room add company6@ 9  100    1    11-20    6
room add company6@ 5  100    1    11-20    6

#cannot add room:
#Error: The room ID already exists in the faculty

room add company1@ 1  40    10   11-16  7

#bad 5
escaper add escaper@@ 4 4
escaper add company1@ 4 4
escaper add escaper1@ 20 2
escaper add escaper1@ 15 11
escaper add escaper1@ 15 0

#good escapers
#           email        faculty   skill_level
escaper add escaper1@    5         5
escaper add escaper2@    5         3
escaper add escaper3@    5         10 
escaper add escaper4@    9         9   
escaper add escaper5@    9         6 
escaper add escaper6@   10         4
escaper add escaper7@    3         5
escaper add escaper8@  15          8
escaper add escaper9@   17         2
escaper add escaper10@   9         1
escaper add escaper11@   10        5
escaper add escaper12@   9         3
escaper add escaper13@   5         2

#bad 1
escaper add escaper7@    5         10

#escaper order bad
escaper order escaper@@  10 5 5-10  8
escaper order escaper17@ 10 5 5-10  7
escaper order escaper1@  10 30 6-10 10
escaper order escaper5@  9 6  3-17 10 
escaper order escaper5@  9 6  3-9  5

#good orders

#day 0 total events 4
#day 1 total events 2
#day 2 total events 2
#day 3 total events 6

escaper order escaper3@ 5   1   0-17  3
escaper order escaper5@ 5    2  0-15  2
escaper order escaper7@ 9    8  1-23  4
escaper order escaper8@ 5    5  2-12  3
escaper order escaper1@ 9   9  0-17  4
escaper order escaper5@ 5    5  0-17   1
escaper order escaper4@  9   5   2-12 2
escaper order escaper12@ 9    6      3-14  3
escaper order escaper6@   9   5     3-14  5
escaper order escaper13@ 5    1      3-17  7
escaper order escaper2@ 5     2         3-17 2
escaper order escaper2@ 5     2     3-18 2  
escaper order escaper9@ 5     1      3-18   2
escaper order escaper11@ 5    1   1-17     2

#bad orders (escaper at room || room reserved ) 3
escaper order escaper13@ 9 5 3-17  5
escaper order escaper10@ 5 5 2-12 8
escaper order escaper1@  9 5  3-14 6


#bad remove company   2
company remove company6@ 
company remove company5@

#good company remove
company remove company7@

#bad remove room      4
room remove 18 8
room remove 9 -1
room remove 9 8

#bad escaper remove 2
escaper remove escaper80@
escaper remove escaper5@6@


#remove escaper 
escaper remove escaper7@

#successfully remove room
room remove 9 8

#add the same room again
room add company5@ 8  36     2    19-24    5

#escaper recommend 1---> new order  room 5 in faculty 9 in day 0 hour 11
escaper recommend escaper5@ 1

#try bad orders first error clinet in room and second error room not available  2
escaper order escaper5@ 9 5 0-11 1
escaper order escaper1@ 9 5 0-11 1 

#escaper recommend 2 after this ->> new order in room 5 faculty 9 in day 0 hour 12
escaper recommend escaper5@ 2

#bad orders first error clinet in room and the second room not available 2
escaper order escaper5@ 9 5 0-11 5
escaper order escaper1@ 9 5 0-11 5

#add new room   (will be used for recommend)
room add company5@ 10  8    12    11-13    6

#add new orders
escaper recommend escaper5@ 11

#try bad orders (first error clinet in room and second error room not available ) 2
escaper order escaper5@ 9 10 1-11 1
escaper order escaper1@ 9 10 1-11 1 


report day

report best

report day

report best

report day

report day

report best



