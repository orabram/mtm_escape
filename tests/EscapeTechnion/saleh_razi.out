Day #0: 6 events in total
The order was made by escaper5@, from the faculty of Chemistry, who is skilled at 6
	It was made for room 5, of company6@ from Chemistry, for the time 11:00
	The room is best for a skill level of 6. 1 people came for the total price of 75
The order was made by escaper5@, from the faculty of Chemistry, who is skilled at 6
	It was made for room 5, of company6@ from Chemistry, for the time 12:00
	The room is best for a skill level of 6. 2 people came for the total price of 150
The order was made by escaper5@, from the faculty of Chemistry, who is skilled at 6
	It was made for room 2, of company1@ from Aerospace Engineering, for the time 15:00
	The room is best for a skill level of 7. 2 people came for the total price of 40
The order was made by escaper3@, from the faculty of Aerospace Engineering, who is skilled at 10
	It was made for room 1, of company1@ from Aerospace Engineering, for the time 17:00
	The room is best for a skill level of 5. 3 people came for the total price of 90
The order was made by escaper5@, from the faculty of Chemistry, who is skilled at 6
	It was made for room 5, of company3@ from Aerospace Engineering, for the time 17:00
	The room is best for a skill level of 6. 1 people came for the total price of 16
The order was made by escaper1@, from the faculty of Aerospace Engineering, who is skilled at 5
	It was made for room 9, of company6@ from Chemistry, for the time 17:00
	The room is best for a skill level of 6. 4 people came for the total price of 400
That is all for day #0!

The system has 18 faculties and in 1 days, they have made a revenue of 771
One of the best faculties - Chemistry, has made a revenue of 625 so far!
One of the best faculties - Aerospace Engineering, has made a revenue of 146 so far!
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!

Day #1: 2 events in total
The order was made by escaper5@, from the faculty of Chemistry, who is skilled at 6
	It was made for room 10, of company5@ from Chemistry, for the time 11:00
	The room is best for a skill level of 6. 11 people came for the total price of 66
The order was made by escaper11@, from the faculty of Biology, who is skilled at 5
	It was made for room 1, of company1@ from Aerospace Engineering, for the time 17:00
	The room is best for a skill level of 5. 2 people came for the total price of 80
That is all for day #1!

The system has 18 faculties and in 2 days, they have made a revenue of 917
One of the best faculties - Chemistry, has made a revenue of 691 so far!
One of the best faculties - Aerospace Engineering, has made a revenue of 226 so far!
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!

Day #2: 2 events in total
The order was made by escaper8@, from the faculty of Materials Engineering, who is skilled at 8
	It was made for room 5, of company3@ from Aerospace Engineering, for the time 12:00
	The room is best for a skill level of 6. 3 people came for the total price of 48
The order was made by escaper4@, from the faculty of Chemistry, who is skilled at 9
	It was made for room 5, of company6@ from Chemistry, for the time 12:00
	The room is best for a skill level of 6. 2 people came for the total price of 150
That is all for day #2!

Day #3: 6 events in total
The order was made by escaper6@, from the faculty of Biology, who is skilled at 4
	It was made for room 5, of company6@ from Chemistry, for the time 14:00
	The room is best for a skill level of 6. 5 people came for the total price of 500
The order was made by escaper12@, from the faculty of Chemistry, who is skilled at 3
	It was made for room 6, of company4@ from Chemistry, for the time 14:00
	The room is best for a skill level of 3. 3 people came for the total price of 54
The order was made by escaper13@, from the faculty of Aerospace Engineering, who is skilled at 2
	It was made for room 1, of company1@ from Aerospace Engineering, for the time 17:00
	The room is best for a skill level of 5. 7 people came for the total price of 210
The order was made by escaper2@, from the faculty of Aerospace Engineering, who is skilled at 3
	It was made for room 2, of company1@ from Aerospace Engineering, for the time 17:00
	The room is best for a skill level of 7. 2 people came for the total price of 30
The order was made by escaper9@, from the faculty of Biomedical Engineering, who is skilled at 2
	It was made for room 1, of company1@ from Aerospace Engineering, for the time 18:00
	The room is best for a skill level of 5. 2 people came for the total price of 80
The order was made by escaper2@, from the faculty of Aerospace Engineering, who is skilled at 3
	It was made for room 2, of company1@ from Aerospace Engineering, for the time 18:00
	The room is best for a skill level of 7. 2 people came for the total price of 30
That is all for day #3!

The system has 18 faculties and in 4 days, they have made a revenue of 2019
One of the best faculties - Chemistry, has made a revenue of 1395 so far!
One of the best faculties - Aerospace Engineering, has made a revenue of 624 so far!
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!

